libraries/api-extractor-model/src/mixins/ApiItemContainerMixin.ts

```diff
@@ -19,6 +19,7 @@ import { InternalError, LegacyAdapters } from '@rushstack/node-core-library';
  */
 export interface IApiItemContainerMixinOptions extends IApiItemOptions {
   members?: ApiItem[];
+  preserveEnumMemberOrder?: boolean;
```

Comment:
Can we rename this to something more generic and not enum-specific, as `ApiItemContainerMixin` doesn't know anything about enums and shouldn't know anything? What about `shouldSortMembers`?